apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/Trace/UnchainedTraceHeaderPropagatingHandler.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Diagnostics.Common
+{
+    /// <summary>
+    /// Class for tracing outgoing HTTP requests and propagating the trace header.
+    /// </summary>
+    /// <remarks>
+    /// Ensures the trace header is propagated in the headers for outgoing HTTP requests and 
```

Comment:
Nit: put the two paragraphs in `<para>` tags - although the second one may go away given the suggestion below.

Reply:
Done, for now, I'm adding a commit just for the nits.